spread.yaml

```diff
@@ -148,6 +148,17 @@ backends:
                 username: test
                 password: ubuntu
 
+    nested:
```

Comment:
I added it only to be able to completely filter the suite and not including it in travis' executions given the time those tests take currently. I didn't know about the `manual: true` setting, applying it to the suite is enough for filtering it, thanks!